app/services/image-processor.service.js

```diff
@@ -24,47 +24,88 @@ async function init() {
 
 init();
 
-async function generateImage(svgHtml, image_dir, imageName, isTeamMessage) {
+async function generateImageTeam(svgHtml, imageName) {
   return new Promise((resolve, reject) => {
     try {
-      wkhtmltoimage.generate(svgHtml, { output: image_dir + imageName }, async function (code, signal) {
-        await base64Img.base64(image_dir + imageName, function (err, base64Bytes) {
+      wkhtmltoimage.generate(svgHtml, { output: config.imageFolder + imageName }, async function (code, signal) {
+        await base64Img.base64(config.imageFolder + imageName, function (err, base64Bytes) {
           var encodedUrl = "data:image/png;base64," + base64Bytes;
-          var base64 = [];
-          if (fs.existsSync(image_dir + imageName)) {
+          if (fs.existsSync(config.imageFolder + imageName)) {
 
-            base64.push({ key: "Email", "encodedUrl": encodedUrl })
+            (async () => {
+              await compress_images(config.imageFolder + imageName, config.compressImageFolder, { compress_force: false, statistic: true, autoupdate: true }, false,
+                { jpg: { engine: 'mozjpeg', command: ['-quality', '60'] } },
+                { png: { engine: 'pngquant', command: ['--quality=20-50'] } },
+                { svg: { engine: 'svgo', command: '--multipass' } },
+                { gif: { engine: 'gifsicle', command: ['--colors', '64', '--use-col=web'] } }, function (error, completed, statistic) {
 
-            if (isTeamMessage) {
-              (async () => {
-                await compress_images(image_dir + imageName, config.compressImageFolder, { compress_force: false, statistic: true, autoupdate: true }, false,
-                  { jpg: { engine: 'mozjpeg', command: ['-quality', '60'] } },
-                  { png: { engine: 'pngquant', command: ['--quality=20-50'] } },
-                  { svg: { engine: 'svgo', command: '--multipass' } },
-                  { gif: { engine: 'gifsicle', command: ['--colors', '64', '--use-col=web'] } }, function (error, completed, statistic) {
+                  if (completed === true) {
```

Comment:
should be  if (completed)

Reply:
no, true it required
because if the image is not compress  then its return false and I only want to process if images are compress